src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1512,6 +1514,7 @@ public abstract partial class TextWriter : System.MarshalByRefObject, System.IDi
         public virtual void Write(uint value) { }
         [System.CLSCompliantAttribute(false)]
         public virtual void Write(ulong value) { }
+        public virtual void Write(ReadOnlySpan<char> source) { }
```

Comment:
since the char[] parameter is called "buffer", I think we should stick with "buffer". Same with WriteLine.

Reply:
I continued with the word mentioned in the approved API. Then likewise should it be `buffer` for `Read/Block` overloads as well, instead of `destination`...?

Reply:
If we agree that as a general guideline, then I will use the same rule for #22408, #22409, etc. as well. Appreciate your inputs.

Reply:
Hi Mandar, is this the only item remaining regarding the PR? Thanks.

Reply:
This one and please see the conversation just above this one over [here](https://github.com/dotnet/corefx/pull/23786#discussion_r139276378). Thanks!!!

Reply:
Yes, we should use "buffer" instead of "destination" in all APIs with "buffer" precedent.

Reply:
@KrzysztofCwalina, what about Stream.Read/Write{Async}?  Those use "buffer" as well, but it was in the context of those methods that we'd settled on "source" and "destination".  Are you suggesting that those change, too?

Reply:
OK< since we already merged some "destination" parameters similar to this one. I am ok with merging this PR and then doing cleanup if we decide it's worth it.